### ISSUE TITLE
SPIKE [settings]: payment methods 1:1 or 1:n display

### ISSUE THREAD

**Issue by frosso** [Automattic/woocommerce-payments#1434]
## Description
In the [new designs for the settings page](paJDYF-1cQ-p2), the intent is to [show one row for the WCPay payment method](https://d.pr/i/dW1ACL), which will include multiple payment methods within it.
At checkout, we will display all the payment methods enabled within WCPay: https://d.pr/i/TE9d5F

## Questions
- Can we accomplish showing one payment method in the WC core settings which enables multiple payment methods at checkout?
  - i.e.: do the payment methods at checkout need to have a 1:1 representation with the payment methods displayed in the WC core settings?

**Comment by frosso** [comment]
I investigated this task. I collected my findings on paJDYF-1cQ-p2#comment-5201 .
To provide a summary - each payment gateway displayed at checkout _does_ need a 1:1 representation on the settings view.
It is however possible to "trick" the `woocommerce_payment_gateways` filter to not add our payment methods when the settings view is displayed.

WC Core modifications would be necessary to accomplish the provided designs.
